tree/dataframe/src/RLoopManager.cxx

```diff
@@ -226,6 +232,54 @@ struct MaxTreeSizeRAII {
    ~MaxTreeSizeRAII() { TTree::SetMaxTreeSize(fOldMaxTreeSize); }
 };
 
+struct DatasetLogInfo {
+   std::string fDataSet;
+   ULong64_t fRangeStart;
+   ULong64_t fRangeEnd;
+};
+
+std::string LogRangeProcessing(const DatasetLogInfo &info)
+{
+   std::stringstream msg;
+   msg << "Processing " << info.fDataSet << ": entry range [" << info.fRangeStart << "," << info.fRangeEnd - 1
```

Comment:
Do you think that including info about the slot index could be useful here as well?

Reply:
why not :D

Reply:
done